src/main/java/lotto/domain/WinningSheet.java

```diff
@@ -4,32 +4,43 @@
 
 public enum WinningSheet {
     FAIL(0, 0),
-    FOURTH(3, 5000),
-    THIRD(4, 50000),
-    SECOND(5, 1500000),
-    FIRST(6, 2000000000);
+    FIFTH(3, 5_000),
+    FOURTH(4, 50_000),
+    THIRD(5, 1_500_000),
+    SECOND(5, 30_000_000),
+    FIRST(6, 2_000_000_000);
 
     private final int matchCount;
-    private final int price;
+    private final int prize;
 
-    WinningSheet(int matchCount, int price) {
+    WinningSheet(int matchCount, int prize) {
         this.matchCount = matchCount;
-        this.price = price;
+        this.prize = prize;
     }
 
-    public static WinningSheet findByMatchCount(int matchCount) {
-        return Arrays.stream(WinningSheet.values())
+    public static WinningSheet findByMatchCount(int matchCount, boolean matchBonus) {
+        WinningSheet findSheet = Arrays.stream(WinningSheet.values())
```

Comment:
```suggestion
        WinningSheet findSheet = Arrays.stream(values())
```
Enum 내부에서의 메서드 호출이라 `values()` 만 사용해도 돼요.